### PR TITLE
[MAINTENANCE] added readme for each lambda

### PR DESCRIPTION
**Pull request by bvolodarskiy**

### CODE REVIEW

**Review by egorodov — APPROVED** [review]
LGTM

File: functions/allure_report/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Allure Report
+
+## Description
+Allure Report Lambda convert GX validations format to Best open-source Test report tool format and generate report
+
+### Libraries and Tools
+great-expectations
```
Comment (a-chumagin) [inline functions/allure_report/README.md:7]: use unordered list format

File: functions/allure_report/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Allure
+awswrangler
+
+### Convert results
+Convert GX validations json results to Allure json result through custom mapper
+Input: Path to json validation result on S3
+Output: allure results in json format
```
Comment (a-chumagin) [inline functions/allure_report/README.md:14]: use line breaks

File: functions/allure_report/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Generate Report
+Generate from Allure json results html page
+Input: Allure results(json)
+Output: Allure report(html+JS)
+
```
Comment (a-chumagin) [inline functions/allure_report/README.md:20]: use line breaks

File: functions/allure_report/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+## Flow
+![Preview Image](https://raw.githubusercontent.com/provectus/data-quality-gate/main/functions/allure_report/allure_report_flow.png)
+
```
Comment (a-chumagin) [inline functions/allure_report/README.md:23]: better to use format `Tux, the Linux mascot`

**Review by a-chumagin — CHANGES_REQUESTED** [review]
I made review for one file but my comments are actual for all new Readme

**Review by oksanagorbachenko — APPROVED** [review]

**Review by a-chumagin — APPROVED** [review]

**Review by a-chumagin — APPROVED** [review]
LGTM

**Pull request merged**